src/LabLog/Controllers/AdminController.cs

```diff
@@ -21,43 +21,25 @@ public class AdminController : Controller
 
         private readonly EventModelContext _db;
         private string _user = "user";
-        private SchoolEventHandler _school;
-        private List<SchoolModel> _schools;
+        private SchoolService _schoolService;
         public AdminController(EventModelContext db)
         {
             _db = db;
-            _school = new SchoolEventHandler(_user, _db); 
-            _schools = _db.Schools.ToList();
+            _schoolService = new SchoolService(db, _user);
         }
 
-        [Route("/")]
+        [Route("/")] //This is temporary. Remove when teacher view is implemented.

```

Comment:
In these types of comments, start with TODO, a lot of IDEs pick it up and flag it in a different colour. Also makes it easier to search for